Improve worker queues overview, other minor improvements (#1411)
astronomer/docs: astro/airflow-alerts.md

### Integrate with SendGrid

> [SendGrid](https://sendgrid.com/) is an email delivery service that's easy to set up for Airflow alerts. A free SendGrid account grants users 40,000 free emails within the first 30 days of an account opening and 100 emails per day after that. This should be enough emails for most alerting use cases.

Reviewer: ```suggestion
[SendGrid](https://sendgrid.com/) is an email delivery service that you can use to configure Airflow alerts. A free SendGrid account grants users 40,000 free emails within the first 30 days of an account opening and 100 emails per day after that. This should be enough emails for most alerting use cases.
```
![image](https://user-images.githubusercontent.com/104205257/199476638-181590f7-b825-415d-ab1a-ec3a33ba10a2.png)
